setup.py

```diff
@@ -110,6 +110,7 @@
         "Intended Audience :: Education",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.10",
```

Comment:
3.9 is not in the trove classifiers as well.

Reply:
Add it.